utils/wandb_logging/log_dataset.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+from pathlib import Path
+
+import yaml
+
+from wandb_utils import WandbLogger
+from utils.datasets import LoadImagesAndLabels
+
```

Comment:
@glenn-jocher All the changes that you made look good. I have changed the import order here because importing `WandbLogger` makes sure that the utils directory is added to the system path, eliminating the need for relative imports.